### ISSUE TITLE
what are all of the controls?

### ISSUE THREAD

**Issue by Kenobork** [theclashman2/GBA-Emulator#11]
Hey guys, so I just found this emulator, and I have no idea what the controls are. I've found start, a and b, but I have no idea what l and r are, can ya'll help me?

**Comment by RSC-Games** [comment]
It took me a while, but I have a full list of controls:

Start: Enter
Select: Shift
A: X
B: Z
L: 1
R: 2
D-Pad: Arrow keys.

Here you go! I hope this helps.

**Comment by Kenobork** [comment]
you are amazing thank you

**Comment by LeviSkies** [comment]
is it possible to set new controls?

**Comment by Darklife12354** [comment]
> is it possible to set new controls?

Yes you want to find the folder **user_scripts** and find a file called **IodineGBAJoyPadGlueCode.js** should be one of the first things you see when you open the file. Then replace the keys with whatever keys you want

**Comment by MrZaza13** [comment]
what is L1
